src/ts/uifactory.ts

```diff
@@ -129,7 +129,7 @@ export namespace UIFactory {
         }),
         new Container({
           components: [
-            new PlaybackToggleButton(),
+            new PlaybackToggleButton({ enterFullscreenOnInitialPlayback: false }),
```

Comment:
why are you not using the config here?

Reply:
Not sure I understand your question. I'm adding the component config here to the component? 🤔